common.blocks/issue/issue.js

```diff
@@ -36,13 +38,21 @@ provide(bemDom.declBlock(this.name, {
         var footer = this._elem('footer');
 
         event.preventDefault();
-        button.setMod('disabled');
 
-        request(button.params.number + '/comments')
-            .then(function(response) { return response.text(); })
-            .then(function(data) {
-                bemDom.append(footer.domElem, data);
-            });
+        if (!this.isCommentsOpened) {
```

Comment:
i think it's better first to check if comments are visible, and then that they are not